roles/install_yamls/README.md

```diff
@@ -57,3 +57,66 @@ The role can be imported and tasks can be executed like this
 
 Tasks are created in a form of: `make_{{ original make target }}`, corresponding to `make target` imported from [a Makefile](https://github.com/openstack-k8s-operators/install_yamls/blob/c8487df41bf9ddefa7989f9384e77ae9720ce9dd/Makefile#L418).
 When task is being executed, it runs corresponding code from the Makefile.
+
+## Make install_yaml Parameter
+
+### module: generate_make_tasks
+
+1. For every makefile target we have a dedicated task file.
+
+```{collapse} Show code
+    ~~~{literalinclude}
+    options:
+      install_yamls_path:
+      description:
+        - Absolute path to install_yamls repository.
+      required: True
+      type: str
+      output_directory:
+      description:
+        - Absolute path to the output directory. It must exists.
+      required: True
+      type: str
+    ~~~
+```
+
+2. Follow below steps to find makefile target dedicated task files
+
+Connect to the controller-0 node and go to ci-framework directory
+```Bash
+  [laptop]$ ssh controller-0
+  [controller-0]$ cd ~/src/github.com/openstack-k8s-operators/ci-framework
+  [controller-0]$ cd ci-framework-data/artifacts/roles/install_yamls_makes/tasks
+```
+
+### Overide install_yaml make parameter
+
+```{collapse} Show code
+    ~~~{literalinclude}
+    MAKE_TMPL = """---
+    - name: Debug make_%(target)s_env
+      when: make_%(target)s_env is defined
+      ansible.builtin.debug:
+        var: make_%(target)s_env
+    - name: Debug make_%(target)s_params
+      when: make_%(target)s_params is defined
+      ansible.builtin.debug:
+        var: make_%(target)s_params
+    - name: Run %(target)s
+      retries: "{{ make_%(target)s_retries | default(omit) }}"
+      delay: "{{ make_%(target)s_delay | default(omit) }}"
+      until: "{{ make_%(target)s_until | default(true) }}"
+      register: "make_%(target)s_status"
+      ci_script:
+        output_dir: "{{ cifmw_basedir|default(ansible_user_dir ~ '/ci-framework-data') }}/artifacts"
+        chdir: "%(chdir)s"
+        script: "make %(target)s"
+        dry_run: "{{ make_%(target)s_dryrun|default(false)|bool }}"
+        extra_args: "{{ dict((make_%(target)s_env|default({})), **(make_%(target)s_params|default({}))) }}"
+        """  # noqa
+    ~~~
```

Comment:
Also, I'm not sure we really want to inject the template in the doc.
We probably should just state something like:
"You can build specific parameters using `make_TARGET_env` and `make_TARGET_dryrun`" and showing a couple of examples.

If people want to know more, they can go in the code directly. Maybe you can point to the file with a link, but I'm not sure it would be really needed.

Reply:
done